google-cloud-firestore/src/main/java/com/google/cloud/firestore/collection/ArraySortedMap.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore.collection;
+
+import com.google.api.core.InternalApi;
+import java.util.*;
```

Comment:
Don't use star imports (applies to multiple files)

Reply:
This is from the original source. I updated the files here.